server.js

```diff
@@ -184,12 +214,47 @@ const authenticateUser = async (req, res, next) => {
   }
 };
 
-app.get('/loginmessage', authenticateUser);
-app.get('/loginmessage', (req, res) => {
-  res.json({secret: 'Login works'})
+
+// Add plant to garden
+app.post("/addplant", authenticateUser);
+app.post("/addplant", async (req, res) => {
+  try {
+  const { plantname, species } = req.body;
+  const accessToken = req.header("Authorization");
+  const user = await User.findOne({accessToken: accessToken});
+  const plants = await new Plant({
+    plantname: plantname,
+    species: species,
+    user: user._id
+  }).save()
+  res.status(200).json({
+    success: true,
+    response: {
+      plants: plants
+    }
+})
+} catch (e) {
+  res.status(500).json({
+    success: false,
+    response: e
+  })
+}
+});
+
+// Get user's plants
+app.get("/plants", authenticateUser);
+app.get("/plants", async (req, res) => {
```

Comment:
Need to sync as route in frontend is
<Route path="/:username/garden" element={<UserGarden />} />